common/lib/capa/capa/inputtypes.py

```diff
@@ -93,12 +93,12 @@ def __init__(self, status, gettext_func=unicode):
         }
         tooltips = {
             # Translators: these are tooltips that indicate the state of an assessment question
-            'correct': _('This is correct.'),
-            'incorrect': _('This is incorrect.'),
-            'partially-correct': _('This is partially correct.'),
-            'unanswered': _('This is unanswered.'),
-            'unsubmitted': _('This is unanswered.'),
-            'queued': _('This is being processed.'),
+            'correct': _('This question is correct.'),
```

Comment:
This seems confusing because it isn't that the question is correct, it is that the answer to the question is correct. Same thing for the others here. I'd run this by someone from doc, e.g. @srpearce.